packages/bigquery/system-test/bigquery.js

```diff
@@ -471,6 +471,18 @@ describe('BigQuery', function() {
         });
       });
 
+      it('should import data from a file via promises', function() {
+        return table.import(file)
+          .then(function(results) {
+            return results[0].promise();
+          })
```

Comment:
Do we need to handle the errors from both of these promises, or does returning the promise to mocha figure that out?